server/src/test/java/org/apache/uniffle/server/StorageCheckerTest.java

```diff
@@ -101,6 +102,8 @@ long getUsedSpace(File file) {
         case "st2":
           switch (callTimes) {
             case 0:
+              result = 200;
+              break;
```

Comment:
Is this change necessary?

See: https://checkstyle.sourceforge.io/config_coding.html#FallThrough